jobs/tendrl-build-triggers.yml

```diff
@@ -100,7 +100,6 @@
     target_repo: 'tendrl-test'
     branch: 'release/1.4.0'
```

Comment:
This should be solved by commit https://github.com/Tendrl/usmqe-centos-ci/pull/1/commits/332712af3cc529db02de25c46541b45ae2f73209.
It lists all branches matching the pattern `{branch}\*`, sort them and take the latest one.